### PR TITLE
Feature/front end bookings overview

### PR DESCRIPTION
**Pull request by JamieKalloe**

### CODE REVIEW

File: src/main/resources/static/js/bookings.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+getBookingsOverview = function() {
+    $.get("/api/bookings/overview", function(bookingData) {
+        for(var index = 0; index < bookingData.length; index++) {
```
Comment (CG-ASaid) [inline src/main/resources/static/js/bookings.js]: foreach misschien?

Comment (CG-ASaid) [reply src/main/resources/static/js/bookings.js]: Obsolete

File: src/main/resources/static/test.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+        $('#table').DataTable({
+            columns: [
+                { "data": "id" },
+                { "data": "guest.lastName" },
+                { "data": "startBooking" },
+                { "data": "endBooking" }
+                    ]
+        });
+        //TODO use it from the external file!
```
Comment (CG-ASaid) [inline src/main/resources/static/test.js]: TODO: remove TODO

Comment (JamieKalloe) [reply src/main/resources/static/test.js]: DONE

**Comment by CG-ASaid** [comment]
toppie

**Pull request merged**